pkg/exec/exec.go

```diff
@@ -36,9 +38,13 @@ func InstallTsh() error {
 	return os.WriteFile(dir+"/tsh", tsh, 0755)
```

Comment:
This might cause issue in m1 where after executing the command it just returns `zsh: killed` because of some gatekeeper issues in macOS. This can be solved by running codesign after binary is copied so binary doesn't get quarantined. `codesign --sign - --force --preserve-metadata=entitlements,requirements,flags,runtime <binary-path>` should do the trick.

Reply:
hmm.. I will check on it. 👍

Reply:
Did you face above issue with a binary built-for arm or x86_64?  We are not planning to build an arm-based binary but relaying on rosetta to the job on m1/2 macs. :crossed_fingers:

Reply:
Nope. There is no issue with x86_64. Using Rosetta would mean people will have to install it. The issue occurs on arm machines only. Even homebrew guys had to implement a workaround. https://github.com/Homebrew/brew/commit/4c19d6717614c1c17fa6c557c9ef41d7ff068c1e
I found the above command from there only 🤪

Reply:
Or installation can be manual. Like it is done for `kubectl` right now. We just check if its installed and ask people to install it of it isn't